e2e/app.e2e-spec.ts

```diff
@@ -23,4 +23,21 @@ describe('Tree App', () => {
     expect(firstAsyncChild.getText()).toEqual('Input Mono');
     expect(page.getLastAsyncChild().getText()).toEqual('Source Code Pro');
   });
+
+  it('Should render tree node with HTML tags', () => {
+    page.navigateTo();
+
+    const firstChildNode = page.getFirstChildNode();
+    expect(browser.isElementPresent(firstChildNode)).toBeTruthy();
+    expect(firstChildNode.getText()).toEqual('Antiqua');
+
+    // verify if the element attributes are same
+    const attrs = {'id': 'antiqua', 'class': 'test'};
+
+    [].forEach.call(Object.keys(attrs), (key) => {
```

Comment:
`Object.keys(attrs).forEach` is a bit easier to read. Also you 100% safe in here calling `forEach` directly